### ISSUE TITLE
I can't use next-page-tester in my test repository

### ISSUE THREAD

**Issue by jefferson-william** [toomuchdesign/next-page-tester#183]
## 💬 Questions and Help

My test project has _styled-component_ and I'm not sure if I configured it correctly to use _next-page-tester_.

**I would like help to run the tests of my project.**

I participated in a discussion on the problem of using _next-page-tester_ in a project with _styled-component_ [here](https://github.com/toomuchdesign/next-page-tester/issues/142#issuecomment-782899351).

I updated all the packages but it still doesn't work.

I created an issue in my repository with more details. Look:

- https://github.com/jefferson-william/next-boilerplate/issues/1

**Steps to reproduce the problem:**

![image](https://user-images.githubusercontent.com/2935122/110157142-9e6ef300-7dc6-11eb-921b-66806cf50021.png)

```bash
git clone https://github.com/jefferson-william/next-boilerplate.git ; cd next-boilerplate
git fetch --all ; git checkout next-page-tester ; yarn install ; yarn test
```

```bash
npx envinfo --preset jest
npx: installed 1 in 1.307s

  System:
    OS: macOS 11.2.1
    CPU: (4) x64 Intel(R) Core(TM) i5-7600K CPU @ 3.80GHz
  Binaries:
    Node: 14.15.0 - ~/.nvm/versions/node/v14.15.0/bin/node
    Yarn: 1.22.10 - ~/.nvm/versions/node/v14.15.0/bin/yarn
    npm: 6.14.8 - ~/.nvm/versions/node/v14.15.0/bin/npm
```

**Comment by Meemaw** [comment]
You need to patch jest, see https://github.com/toomuchdesign/next-page-tester#optional-patch-jest for details

**Comment by toomuchdesign** [comment]
Hi @jefferson-william,
would you mind setting up a minimal repo isolating the issue you're experiencing?

Bonus: would you make sure that Jest is able to handle async `beforeEach` without calling `done` callback?
Greetings

**Comment by toomuchdesign** [comment]
Fixed by #186
